lib/rubocop/cop/layout/space_in_lambda_literal.rb

```diff
@@ -3,8 +3,8 @@
 module RuboCop
   module Cop
     module Layout
-      # This cop checks for spaces between -> and opening parameter
-      # parenthesis in lambda literals.
+      # This cop checks for spaces between `->` and opening parameter
+      # `(` in lambda literals.
```

Comment:
Here maybe we have it say "parenthesis (`(`)", as in the cop description we can be more descriptive.